packages/flutter/test/widgets/reorderable_list_test.dart

```diff
@@ -267,6 +267,103 @@ void main() {
     await tester.pumpAndSettle();
     expect(getItemFadeTransition(), findsNothing);
   });
+
+  group('$ReorderableDragStartListener', () {
```

Comment:
Please don't use `$Classname` for the name here, as it makes it hard to run the test or group using the `--plain-name` parameter to `flutter test`. Just `ReorderableDragStartListener` would be better.